src/sentry/api/bases/organization_events.py

```diff
@@ -253,6 +253,8 @@ def get_event_stats_data(
                     "event_count": "count()",
                     "epm()": "epm(%d)" % rollup,
                     "eps()": "eps(%d)" % rollup,
+                    "tpm()": "tpm(%d)" % rollup,
```

Comment:
Can we add a comment in addition to the above to mention that we need to add the rollups to generate the correct `events-stats`?